src/collection-browser.ts

```diff
@@ -617,7 +619,12 @@ export class CollectionBrowser
         @displayModeChanged=${this.displayModeChanged}
         @titleLetterChanged=${this.titleLetterSelected}
         @creatorLetterChanged=${this.creatorLetterSelected}
+        .showLoansTopBar=${this.isLoansTab}
       >
+        <slot
+          name="loans-tab-top-bar-options-slot"
+          slot="loans-tab-top-bar-options-slot"
+        ></slot>
```

Comment:
Instead of saying `top-bar` everywhere, we should `filter-bar` to make consistent between offshoot and collection-browser.